app/src/main/res/drawable/ic_qrcode_scan_black.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="@color/gray_dark"
```

Comment:
`fillColor` and `filename` needs to be in sync so that filenames are not misleading. Correct it as per the current use case. Do we need a new `drawable` or one is suffice. Please Check

Reply:
I have updated it.